app/channels/application_cable/channel.rb

```diff
@@ -0,0 +1,4 @@
+module ApplicationCable
```

Comment:
Whats this?

Reply:
This is where we define the server-side connection logic for when a user's browser subscribes to a Websocket connection channel.

Reply:
This is our base class for all channels. This will be inherited by all channels. This is where you can define common methods that will be helpful across any or all of your channels. Channels are a named, organized way to define behavior with both server and client methods, by which a client browser can subscribe and then communicate both ways.